crawl-ref/source/makeitem.cc

```diff
@@ -647,6 +643,7 @@ bool is_missile_brand_ok(int type, int brand, bool strict)
     case SPMSL_EXPLODING:
         return type == MI_TOMAHAWK;
     case SPMSL_STEEL: // deliberate fall through
+	return false;
```

Comment:
I used one tab to adjust the spacing. But it caused a checkwhite=1 error. What's the problem?

Reply:
You need to use four spaces for each tab. So in this case you want ```
case SPMSL_STEEL:
        return false;```
with eight spaces (two indents).

Also, a note about using git that might help you. If you're using the command line, you can make your fix and then run `git add -A; git commit --amend`, then `git push --force` to fix this without making a new PR or additional unnecessary commits. (It will update your latest commit and then push the updated commit to the PR, overwriting the outdated commit.)

Reply:
```
 case SPMSL_STEEL:
         return false;
 ```
1. one tap change to eight space
2. use git command line
3. type `git add -A` and `git commit --amend`
right?

Reply:
Correct, then finally `git push --force`

Reply:
Thank you very much for your kind explanation. I solved the problem.

Reply:
No problem :)